boot/zephyr/CMakeLists.txt

```diff
@@ -191,6 +191,11 @@ elseif(CONFIG_BOOT_SIGNATURE_TYPE_RSA)
   # Use mbedTLS provided by Zephyr for RSA signatures. (Its config file
   # is set using Kconfig.)
   zephyr_include_directories(include)
+  if(CONFIG_BOOT_ENCRYPT_RSA)
+    set_source_files_properties(${BOOT_DIR}/bootutil/src/encrypted.c
+    PROPERTIES
```

Comment:
Sorry for the nit pick, but I think this will look better with these additional lines indented, otherwise they look more like additional top-level commands). I think the indentation just got dropped from the comment.